preload.js

```diff
@@ -502,7 +510,7 @@ window.pubkeyPattern = /@[a-fA-F0-9]{64,66}\b/g;
 window.lokiFeatureFlags = {
   multiDeviceUnpairing: true,
   privateGroupChats: true,
-  useSnodeProxy: true,
+  useSnodeProxy: false,
```

Comment:
probably don't want this in the final version

Reply:
would something like that be good? 

```
window.lokiFeatureFlags = {
  multiDeviceUnpairing: true,
  privateGroupChats: true,
  useSnodeProxy: !process.env.USE_STUBBED_NETWORK,
  useSealedSender: true,
};
```

Reply:
Yes, that looks fine to me